dimes/dimes.py

```diff
@@ -110,19 +110,19 @@ def finalize_plot(self):
         if not self.is_finalized:
```

Comment:
Suggestion to create a new variable to replace `subplot_number - 1` on lines 104-106.